fusion/example_video.py

```diff
@@ -0,0 +1,270 @@
+# Given a video sequence we want to run NNRT to extract motion
+# Usage python3 example_video.py <path to frames>
+
+import os
+import open3d as o3d
+import torch
+import numpy as np
+
+from utils import image_proc
+from model.model import DeformNet
+from model import dataset
+
+import utils.utils as utils
+import utils.viz_utils as viz_utils
```

Comment:
I split up the code in `utils.viz_utils` between different files in `utils.visualization`